### DISCUSSION TITLE
Stats component with rows

### DISCUSSION THREAD

**Discussion by fini** [saadeghi/daisyui#879] [Q&A] [answered]
Is there a good way to combine stats-vertical and stats-horizontal?
I have 4 columns, that I would like to break into 2x2 on smaller screens, I can make it sort of work by adding grid-cols-2 to the main div, but it breaks borders a bit, so would like to know if there is a nicer method.

**Comment by saadeghi** [comment]
You mean there are 4 steps and you want 2 of them to go to the second row?

**Reply by fini** [reply]
Yeah four div.stat and would like them to collapse to two on each row on smaller devices.

**Comment by emek-lu** [comment]
I'm looking for something similar. I have a page with a max width with a stats component inside. When there are too many div.stat-items I want them to appear on multiple rows. At the moment it either grows longer than the containing div, or there's a vertical scroll (if I put a max-width on div.stats). None of these options looks very good. 
I have an example here: https://play.tailwindcss.com/HFIrgBnfIG

Is there a way to make it overflow on multiple rows?

**Comment by saadeghi** [comment, marked as answer]
It's a grid and it's not meant to wrap to a second row.  
You can change it to flex using `inline-flex` class: https://play.tailwindcss.com/sKQNObqZQ1

But it won't look good of course because the item that won't fit will go to the second row and there will be no way to put a separator line between them automatically.
